src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -2060,6 +2072,16 @@ Query the `K8sContainerSample` event for container data:
       </td>
     </tr>
 
+    <tr>
+      <td>
+        `restartCountDelta`
+      </td>
+
+      <td>
+        Delta change of number of times the container has been restarted (requires k8s 1.23+).
```

Comment:
```suggestion
        Delta change of number of times the container has been restarted.
```

I think this metric does not require k8s 1.23+. Its value is got from `restartCount`.